.github/workflows/integrate.yaml

```diff
@@ -62,6 +62,8 @@ jobs:
             provider: microk8s
             channel: 1.22/stable
             charmcraft-channel: latest/candidate
+      # https://github.com/charmed-kubernetes/actions-operator/issues/27
+      - run: sudo snap refresh charmcraft --channel latest/candidate
```

Comment:
I think [this is implemented](https://github.com/charmed-kubernetes/actions-operator/pull/28)

Reply:
Thank you